tot/socket/mock.socketer_test.go

```diff
@@ -4,8 +4,8 @@ import (
 	"net/url"
 	"sync"
 
-	logfmt "github.com/mkenney/go-log-fmt"
-	log "github.com/sirupsen/logrus"
+	"github.com/bdlm/log"
+	"github.com/bdlm/logfmt"
```

Comment:
could not import github.com/bdlm/logfmt (cannot find package "github.com/bdlm/logfmt" in any of:
	/usr/local/go/src/github.com/bdlm/logfmt (from $GOROOT)
	/tmp/tmp.gy1zH1yeLJ/src/github.com/bdlm/logfmt (from $GOPATH))